bigbluebutton-html5/private/locales/en.json

```diff
@@ -255,5 +257,7 @@
     "app.error.leaveLabel": "Log in again",
     "app.guest.waiting": "Waiting for approval to join",
     "app.notification.recordingStart": "This session is now being recorded",
-    "app.notification.recordingStop": "This session is not being recorded anymore"
+    "app.notification.recordingStop": "This session is not being recorded anymore",
+    "app.video.joinVideo": "Cam off",
```

Comment:
@capilkey could you suggest a better label for the start / stop sharing a webcam buttons?
Maybe "Start video" "Stop video"?

Reply:
I'll leave it as Start video/Stop video while I attend to those review requests. If anyone has any suggestions later, just me know.